zipkin-lens/src/components/DiscoverPage/DiscoverPage.jsx

```diff
@@ -125,7 +126,23 @@ const propTypes = {
   setLimitCondition: PropTypes.func.isRequired,
 };
 
-const DiscoverPageImpl = ({
+const DiscoverPageHeader = () => {
```

Comment:
How about splitting files for each components?
This is not a strong opinion, but other components is in the individual file..